### DISCUSSION TITLE
milvus GPU无法调用 cpu可以运行

### DISCUSSION THREAD

**Discussion by xleiz** [milvus-io/milvus#15973] [Q&A and General discussion] [answered]
启用GPU后，在建立索引时会报错，查看日志如下：
Faiss assertion 'err__ == cudaSuccess' failed in void faiss::gpu::runL2Norm(faiss::gpu::Tensor<T, 2, true, IndexType>&, bool, faiss::gpu::Tensor<float, 1, true, IndexType>&, bool, cudaStream_t) [with T = float; TVec = float4; IndexType = int; cudaStream_t = CUstream_st*] at gpu/impl/L2Norm.cu:292; details: CUDA error 209 no kernel image is available for execution on the device

是cuda版本不匹配吗，我用的是milvus1.1.1的docker镜像，很多个cuda版本都试过了，还是报错

**Comment by xleiz** [comment]
还是说faiss不支持16.04的ubuntu系统？

**Reply by caobinbc** [reply]
解决了吗？  最终使用的是那个cuda版本？  我是用milvus1.1.1的GPU版本，cuda 11.4，Tesla K80也是报这个错误

**Comment by yhmo** [comment, marked as answer]
可能是显卡太新了，看一下这个：

https://github.com/milvus-io/milvus/discussions/15746

**Reply by xleiz** [reply]
确实 我是30系的显卡 1.x好像不支持。。。

**Reply by xleiz** [reply]
谢谢您的解答！
